uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Form/Search/StaffDirectory.php

```diff
@@ -118,14 +128,47 @@ private function buildWhere() {
         case 'select_staff':
           $this->setQueryConditionForSelectStaff($value);
           break;
+        case 'name':
+          $this->where[] = CRM_Contact_BAO_Query::buildClause($alias, 'LIKE', "%{$value}%");
+          break;
+        case 'job_title':
+          $this->setQueryConditionForJobDetailsFields($alias, 'LIKE', "%{$value}%");
+          break;
+        case 'department':
+        case 'location':
+          $this->setQueryConditionForJobDetailsFields($alias, '=', "{$value}");
+          break;
         default:
           $this->where[] = CRM_Contact_BAO_Query::buildClause($alias, $op, $value);
       }
     }
 
+    if (!empty($this->jobDetailsCondition)) {
+      $jobDetailsCondition =  implode(' AND ', $this->jobDetailsCondition);
```

Comment:
There seems to be double space between `=` and `implode` 🔍. Can you confirm this please?